MetaMask/metamask-extension
# Keyring bridge updates

## Explanation



It just updates keyring packages, but it should not change any existing features.

- Updates the Ledger and Trezor keyrings, which introduce breaking changes.
- Allows injecting override for the whole keyring array.

## Manual Testing Steps



For both Trezor and Ledger:
- Add new address from the device
- Open Test Dapp
- Sign a transaction
- Sign a message
- Sign a personal message
- Sign typed data v4

This should ensure that all existing features are still working.

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
